extension/content/customPersonaEditor.js

```diff
@@ -243,14 +254,14 @@ var CustomPersonaEditor = {
     PersonaService.changeToPersona(this.customPersona);
     switch (PersonaService.appInfo.ID) {
       case PersonaService.THUNDERBIRD_ID:
-	Cc['@mozilla.org/appshell/window-mediator;1'].
+    Cc['@mozilla.org/appshell/window-mediator;1'].
```

Comment:
I am not sure why it is happening but Github Desktop shows they have the same indent. But on Github.com the lines are misaligned.

Actually the file is already contains too many formatting issues. So it has to be prettified all together.